src/main/java/duke/task/Todo.java

```diff
@@ -0,0 +1,63 @@
+package duke.task;
+
+
+import duke.exception.DukeException;
+
+/**
+ * Represents a todo task.
+ */
+public class Todo extends Task {
+    private static String parseTodo(String task) throws DukeException {
+        String todoTask = task.trim();
+        if (todoTask.isEmpty()) {
+            throw new DukeException("Please enter valid deadline: Do not leave it empty");

```

Comment:
I don't understand when this is thrown... A todo task should have no deadline anyway

Reply:
Thanks for catching my mistake! It should be todo, and it is thrown when the person types todo but did not give a description.